src/_tests/fixtures/38979/mutations.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "query": "mutation($input: AddLabelsToLabelableInput!) { addLabelsToLabelable(input: $input) { clientMutationId } }",
+    "mutation": "mutation ($input: AddLabelsToLabelableInput!) {\n  addLabelsToLabelable(input: $input) {\n    __typename\n  }\n}\n",
```

Comment:
We don't use `clientMutationId` or `__typename` (or any part of the result, it might as well be void) this is just a workaround: selection sets can't be empty. `__typename` is even more generic: belongs to every GraphQL object.